### ISSUE TITLE
Hello world does not work OOTB

### ISSUE THREAD

**Issue by rvansa** [Hyperfoil/qDup#129]
Going through the hello world example I am blocked as my hostname does not resolve to IP. The previous "ensure you can ssh..." uses localhost, then the example uses hostname instead.

**Comment by johnaohara** [comment]
@rvansa what os version are you running this on? we have seen issues with the available tsl ciphers on recent fedora releases

**Comment by johnaohara** [comment]
@rvansa please can you verify if creating a new FIPS compliant key fixes the issue for you? https://github.com/Hyperfoil/qDup/blob/master/docs/ssh-config.adoc#1-problems-with-qdup-and-ssh

**Comment by rvansa** [comment]
@johnaohara I'll try that, thanks.

Btw. it seems there's another issue with the example command (not SSL-related):
```
jbang ./docs/examples/runQDup.java -S GREETING='qDup ROCKS!!' -T
11:58:09.087 [main] ERROR io.hyperfoil.tools.qdup.JarMain - The option 'T' was specified but an option from this group has already been selected: 'B'
org.apache.commons.cli.AlreadySelectedException: The option 'T' was specified but an option from this group has already been selected: 'B'
```

**Comment by johnaohara** [comment]
> 11:58:09.087 [main] ERROR io.hyperfoil.tools.qdup.JarMain - The option 'T' was specified but an option from this group has already been selected: 'B'
> org.apache.commons.cli.AlreadySelectedException: The option 'T' was specified but an option from this group has already been selected: 'B'
> ```
@rvansa ah that option group check was added after the guide, will need to review the guide in light of the change. Thanks for spotting!

**Comment by rvansa** [comment]
@johnaohara Yep, the ECDSA key fixed the problem. qDup should autodetect this issue and print a hint when it happens.

**Comment by johnaohara** [comment]
@rvansa please could you open a new issue re auto-detecting when there are no suitable keys available?

**Comment by rvansa** [comment]
https://github.com/Hyperfoil/qDup/issues/130
